xyo-android-library/src/main/java/network/xyo/sdk/XyoBleClient.kt

```diff
@@ -30,20 +30,20 @@ class XyoBleClient(
     scan: Boolean
 ) : XyoClient(relayNode, procedureCatalog, autoBoundWitness) {
 
-    override var autoBridge: Boolean = false
-    override var acceptBridging: Boolean = false
+    override var autoBridge: Boolean = true
+    override var acceptBridging: Boolean = true
 
     override var deviceCount = 0
     override var xyoDeviceCount = 0
     override var nearbyXyoDeviceCount = 0
 
-    var supportBridgeX = false
-    var supportSentinelX = false
-    var minimumRssi = -70
+    var supportBridgeX = true
+    var supportSentinelX = true
+    var minimumRssi = -95
 
-    val minBWTimeGap = 10 * 1000
+    val minBWTimeGap = 10 * 900
```

Comment:
Did you find in your tests that the bound witnesses were too frequent where some may fail?

Reply:
Only in bluetooth balanced mode or low power mode. Using lower latency works great, but some BW still failing. Adding low latency mode the number of failed bw in higher frequency bw interaction decreases.